CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Change Log
 
+## 0.18.1 - 2021-08-11
+### [Fixed](https://github.com/microsoft/vscode-cosmosdb/milestone/32?closed=1)
+- Fixed the ETIMEDOUT issue "...when signing the PostgreSQL Flexible server on another virtual machine"
```

Comment:
I don't think you have to quote the issue title exactly - something like this would be good:

```suggestion
- Fixed the ETIMEDOUT issue when signing into a PostgreSQL Flexible server on multiple machines
```